CONTRIBUTING.md

```diff
@@ -59,6 +59,14 @@ git clone https://github.com/<UserName>/machine-learning.git
 
 Before you start, you will need [Python](https://wiki.python.org/moin/BeginnersGuide/Download) and [Conda](https://docs.anaconda.com/anaconda/install/) on your computer.
 
+TIPS: Don't forget to add `Python` and `conda` to `PATH`. Like this:
```

Comment:
Let's not do the windows doc like this.

1. create two sections for different OS in `Notes for contributors`  1. Mac, 2. Windows
2. Keep the current instruments inside the 1. Mac section.
3. create new content in windows section.